### PR TITLE
Update POC

### PR DESCRIPTION
**Pull request by claucece**

### CODE REVIEW

File: poc/oprf.sage [hunk 1]
```diff
@@ -103,13 +103,6 @@ class GroupNISTCurve(Group):
             y = -y
         return self.curve(self.F(x), self.F(y))
 
-    def serialize_scalar(self, scalar):
```
Comment (chris-wood) [inline poc/oprf.sage:118]: Why did we remove these? Aren't they needed?

Comment (claucece) [reply poc/oprf.sage:118]: They were not used anywhere, not sure why they were there..

**Comment by claucece** [comment]
@chris-wood all methods implemented now. There are some todos, I'll solve tomorrow. There is a really weird python error with `expand_message_xof`, which idk why is occurring. If you can, please, check it.

**Comment by claucece** [comment]
@chris-wood solved the bug now. Except for a check I want to do tomorrow and some recfator, this is now done.

**Pull request merged**